src/runtime/opencl/opencl_module.cc

```diff
@@ -87,9 +87,9 @@ class OpenCLModuleNode : public ModuleNode {
   }
 
   // Initialize the programs
-  void Init() {
+  void Init(const std::vector<std::string>& device_types, const std::string& platform_name) {
     workspace_ = cl::OpenCLWorkspace::Global();
-    workspace_->Init();
+    workspace_->Init(device_types, platform_name);
```

Comment:
The current choice of platform is mutually exclusive, so that once we load a sdaccel file, it is no longer possible to load an OpenCL GPU module due to the single global singleton being used in here.